op-batcher/batcher/channel.go

```diff
@@ -66,7 +66,6 @@ func (c *channel) TxFailed(id string) {
 // resubmitted.
 // This function may reset the pending channel if the pending channel has timed out.
 func (c *channel) TxConfirmed(id string, inclusionBlock eth.BlockID) (bool, []*types.Block) {
-	c.metr.RecordBatchTxSubmitted()
```

Comment:
I think this should indeed have been a `c.metr.RecordBatchTxSuccess()`. Otherwise, that function is never called.